### PR TITLE
Created a new demo showing how the difference of ephemeral vs persistent storage.

### PR DESCRIPTION
**Pull request by aeroniero33**
https://www.katacoda.com/aeroniero/scenarios/ephemeral

### CODE REVIEW

File: ephemeral/002-deploy-pods-volumes.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+> Usually takes ~30 seconds to come up after it's been created.
+
+You can also confirm StorageOS provisioned the volume with the CLI:
+
+`kubectl exec -ti cli -n kube-system -- storageos get volume`{{execute}}
+
+The link between the PersistentVolumeClaim and the pod comes from the pod
+referencing the PersistentVolumeClaim in the pod's yaml definition.
+
+Run the command to see the ephemeral mysql pod definition:
```
Comment (Arau) [inline ephemeral/002-deploy-pods-volumes.md:26]: People can `cat` the files if they want. Let try to keep the steps small and concise. If someone wants to check, they can look into it. 

If we want to compare things, then we should explicitly show it. I would create another step that just shows in the scenario that one has a pvc and the other doesn't

File: ephemeral/003-create-data.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+Check your commands succeeded:
+
+`SELECT * FROM FRUIT;`{{execute}}
+
+Quit the MySQL container:
+
+`\q`{{execute}}
+
+The continue with the persistent one:
```
Comment (Arau) [inline ephemeral/003-create-data.md:33]: s/The/Then/

File: ephemeral/004-redeploy-pods.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+`kubectl exec mysql-ephemeral -- mysql -e 'show databases;'`{{execute}}
+
+`kubectl exec mysql-persistent -- mysql -e 'show databases;'`{{execute}}
+
+As you can see, the database was lost in the mysql ephemeral pod but still exists in the mysql persisted one and we can find that the data is there as well:
+
+`kubectl exec mysql-persistent -- mysql -e 'use shop; select * from FRUIT;'`{{execute}}
+
+Using persistent storage provisioned by StorageOS can persist data even after pod failover!
```
Comment (Arau) [inline ephemeral/004-redeploy-pods.md:31]: This is nice, however this is not a failover.

"Using persistent storage provisioned by StorageOS makes your data safe between Pod restarts, even if the Pod starts in a different node".

File: ephemeral/002-deploy-pods-volumes.md (hunk 1 above)
Comment (aeroniero33) [reply ephemeral/002-deploy-pods-volumes.md]: I left only 2 cats inside and described what the difference is.

**Pull request merged**